lib/pages/my_reports_pages/my_reports_page.dart

```diff
@@ -1,925 +1,84 @@
-                    );
-                  })),
-
-          StreamBuilder<bool>(
-              stream: loadingStream.stream,
-              initialData: true,
-              builder: (BuildContext context, AsyncSnapshot<bool> snapLoading) {
-                if (snapLoading.data == true) {
-                  return Container(
-                    child: Center(
-                      child: Utils.loading(true),
-                    ),
-                  );
-                }
-                return Container();
-              }),
-
-          Container(
-            child: Card(
-              margin: EdgeInsets.all(0),
-              shape: RoundedRectangleBorder(
-                  borderRadius: BorderRadius.circular(0)),
-              color: Colors.white,
-              elevation: 2,
-              child: SafeArea(
-                bottom: false,
-                child: Column(
-                  mainAxisSize: MainAxisSize.min,
-                  children: <Widget>[
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                      children: <Widget>[
-                        IconButton(
-                          onPressed: () {
-                            Navigator.pop(context);
-                          },
-                          icon: Style.iconBack,
-                        ),
-                        Style.title(
-                            MyLocalizations.of(context, 'your_reports_txt'),
-                            fontSize: 16.0),
-                        SizedBox(
-                          width: 40,
-                        )
-                      ],
-                    ),
-                    Container(
-                      margin: EdgeInsets.only(top: 15, bottom: 16),
-                      width: double.infinity,
-                      child: MaterialSegmentedControl(
-                        children: _children,
-                        selectionIndex: _currentIndex,
-                        borderColor: Style.colorPrimary,
-                        selectedColor: Style.colorPrimary,
-                        unselectedColor: Colors.white,
-                        borderRadius: 5.0,
-                        onSegmentChosen: (dynamic index) {
-                          _onItemTapped(index);
-                        },
-                      ),
-                    ),
-                  ],
-                ),
-              ),
-            ),
+    return DefaultTabController(
+      length: 3,
+      child: Scaffold(
+        appBar: AppBar(
+          backgroundColor: Colors.white,
+          bottom: TabBar(
+            tabs: [
+              Tab(
+                icon: Image.asset(
+                  'assets/img/ic_my_reports_mosquito.png',
+                  width: 24,
+                  height: 24,
+                  filterQuality: FilterQuality.high),
+                text: MyLocalizations.of(context, 'single_mosquito') ?? 'Mosquito'),
```

Comment:
Isn't there already a fallback to english for translations?

Reply:
yes, but https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/blob/f71905e91e2fb0084f83387bb5328930baa0e4a6/lib/utils/MyLocalizations.dart#L46 has a datatype of String? instead of String so it confused me, I've modified it to not be nullable, as it shouldn't be